docker/java/spring/maven-package.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -x
+JAVA_AGENT_BUILT_VERSION=${1}
+
+ARTIFACT_ID=elastic-apm-agent
+
+if [ -z "${JAVA_AGENT_BUILT_VERSION}" ] ; then
+  cd /agent/apm-agent-java
+  mvn -q --batch-mode install -DskipTests \
+      -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+
+  export JAVA_AGENT_BUILT_VERSION=$(mvn -q -Dexec.executable="echo" -Dexec.args='${project.version}' --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
```

Comment:
Is there a way to not specify the version of the plugin? It just seems a bit brittle.

Reply:
Do you mean `org.codehaus.mojo:exec-maven-plugin:1.3.1:exec`? If so, it's safety using the specific version to ensure there are no regressions in the plugin itself with new versions.

Reply:
Ah, gotcha